cdap-watchdog/src/main/java/io/cdap/cdap/logging/ErrorLogsClassifier.java

```diff
@@ -219,6 +249,50 @@ private void populateResponse(IThrowableProxy throwableProxy,
     }
   }
 
+  @Nullable
+  private ErrorClassificationResponse findMatchingRule(CloseableIterator<LogEvent> logIter) {
+    while (logIter.hasNext()) {
+      ILoggingEvent logEvent = logIter.next().getLoggingEvent();
+      IThrowableProxy throwableProxy = logEvent.getThrowableProxy();
+      while (throwableProxy != null) {
+        for (ErrorClassificationRule rule : ruleList) {
+          if (throwableProxy.getClassName().equals(rule.getExceptionClassName())
```

Comment:
Make it regex as well. It would allow you to match multiple classes without rule duplication. And literal regex is very cheap, so it won't affect performance much

Reply:
done.